autoinstall_templates/sample.seed

```diff
@@ -117,6 +117,10 @@ d-i pkgsel/include string ntp ssh wget
 # Note: options passed to the installer will be added automatically.
 d-i debian-installer/add-kernel-opts string $kernel_options_post
 
+# Grub settings
+d-i grub-installer/grub2_instead_of_grub_legacy boolean true
```

Comment:
Since which debian/ubuntu version does this option exists ? (and which version is affected ?)
Do you know if it will work on arches where grub2 isn't used ?